modules/core/src/main/resources/scalafix-migrations.conf

```
@@ -29,6 +29,12 @@ migrations = [
     newVersion: "0.8.0",
     rewriteRules: ["github:spotify/scio/MigrateV0_8?sha=v0.8.0"]
   },
+  {
+    groupId: "com.twilio",
+    artifactIds: ["sbt-guardrail"],
+    newVersion: "0.59.0.1",
+    rewriteRules: ["https://raw.githubusercontent.com/blast-hardcheese/guardrail-scalafix-rules/master/rules/src/main/scala/fix/GuardrailScalaResponseTypes.scala"]
```

Scala Steward can also link to the upgrade notes if they are included here:
```suggestion
    rewriteRules: ["https://raw.githubusercontent.com/blast-hardcheese/guardrail-scalafix-rules/master/rules/src/main/scala/fix/GuardrailScalaResponseTypes.scala"],
    doc: "https://github.com/twilio/guardrail/blob/master/MIGRATING.md#0590-may-contain-type-and-package-naming-changes-that-will-require-changes-in-consuming-code"
```